app/scripts/models/auth_brokers/mixins/channel.js

```diff
@@ -17,11 +17,48 @@ define([
   'underscore',
   'lib/promise'
 ], function (_, p) {
+  function doIt(strategy, channel, message, data) {
+    var boundStrategy = strategy.bind(channel);
```

Comment:
is it possible to have better naming for these? 
`boundStrategy` is confusing, is this a `channelAction` or something?

Same with `doIt` ?